events/nats.go

```diff
@@ -45,6 +45,13 @@ type NatsJetstream struct {
 	subscriberCh  MsgCh
 }
 
+func (n *NatsJetstream) streamReplicas() int {
```

Comment:
We'll need a way to have 0 replicas set on the stream - for the sandbox use case where NATS is setup in the unclustered mode. Thoughts on how that could be done?